warehouse/oso_sqlmesh/models/intermediate/events/int_events__github.sql

```diff
@@ -7,204 +7,149 @@ MODEL (
+    actor_login AS actor_name,
+    actor_id::TEXT AS actor_id,
+    'GIT_USER' AS actor_type
   FROM oso.stg_github__issues
-  WHERE
-    event_time BETWEEN @start_dt AND @end_dt
-), github_pull_requests AS (
-  /* noqa: ST06 */
+  WHERE event_time BETWEEN @start_dt AND @end_dt
+
+  UNION ALL
+
   SELECT
-    event_time AS "time",
+    event_time,
     type AS event_type,
     id::TEXT AS event_source_id,
-    'GITHUB' AS event_source,
-    STR_SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(1)] AS to_name,
-    STR_SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(0)] AS to_namespace,
-    'REPOSITORY' AS to_type,
-    repository_id::TEXT AS to_artifact_source_id,
-    actor_login AS from_name,
-    actor_login AS from_namespace,
-    'GIT_USER' AS from_type,
-    actor_id::TEXT AS from_artifact_source_id,
-    1::DOUBLE AS amount
+    repository_name,
+    repository_id,
+    actor_login AS actor_name,
+    actor_id::TEXT AS actor_id,
+    'GIT_USER' AS actor_type
   FROM oso.stg_github__pull_requests
-  WHERE
-    event_time BETWEEN @start_dt AND @end_dt
-), github_pull_request_merge_events AS (
-  /* noqa: ST06 */
+  WHERE event_time BETWEEN @start_dt AND @end_dt
+
+  UNION ALL
+
   SELECT
-    event_time AS "time",
+    event_time,
     type AS event_type,
     id::TEXT AS event_source_id,
-    'GITHUB' AS event_source,
-    STR_SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(1)] AS to_name,
-    STR_SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(0)] AS to_namespace,
-    'REPOSITORY' AS to_type,
-    repository_id::TEXT AS to_artifact_source_id,
-    actor_login AS from_name,
-    actor_login AS from_namespace,
-    'GIT_USER' AS from_type,
-    actor_id::TEXT AS from_artifact_source_id,
-    1::DOUBLE AS amount
+    repository_name,
+    repository_id,
+    actor_login AS actor_name,
+    actor_id::TEXT AS actor_id,
+    'GIT_USER' AS actor_type
   FROM oso.stg_github__pull_request_merge_events
-  WHERE
-    event_time BETWEEN @start_dt AND @end_dt
-), github_stars_and_forks AS (
-  /* noqa: ST06 */
+  WHERE event_time BETWEEN @start_dt AND @end_dt
+
+  UNION ALL
+
+  /* Stars and Forks */
   SELECT
-    created_at AS "time",
+    created_at AS event_time,
     type AS event_type,
     id::TEXT AS event_source_id,
+    repository_name,
+    repository_id,
+    actor_login AS actor_name,
+    actor_id::TEXT AS actor_id,
+    'GIT_USER' AS actor_type
+  FROM oso.stg_github__stars_and_forks
+  WHERE created_at BETWEEN @start_dt AND @end_dt
+), 
+
+parsed_events AS (
+  SELECT
+    event_time AS time,
+    UPPER(event_type) AS event_type,
+    event_source_id::TEXT AS event_source_id,
     'GITHUB' AS event_source,
-    STR_SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(1)] AS to_name,
-    STR_SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(0)] AS to_namespace,
-    'REPOSITORY' AS to_type,
+    LOWER(STR_SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(1)])
```

Comment:
```suggestion
    LOWER(SPLIT(REPLACE(repository_name, '@', ''), '/')[@array_index(1)])
```

A bit confusing, but by default the parser uses `duckdb` for the dialect so the previous iteration of this actually transpiled `STR_SPLIT` into `SPLIT` from duckdb to trino but since you changed this to be a trino dialect this needs to be `SPLIT` here.